web-ui/view/git_forge.ml

```diff
@@ -15,6 +15,16 @@ module type View = sig
 
   val rebuild_fail_message : int -> [> Html_types.div ] Tyxml_html.elt
 
+  val cancel_success_message_v1 :
+    Client.job_info list -> ([> `Success ] * string) list
+
+  val cancel_fail_message_v1 : int -> ([> `Fail ] * string) list
+
+  val rebuild_success_message_v1 :
+    Client.job_info list -> ([> `Success ] * string) list
+
+  val rebuild_fail_message_v1 : int -> ([> `Fail ] * string) list
```

Comment:
I'm not a big fan of the naming scheme.... I'll explain in the general review message.

Reply:
I'm doing this to accomplish the following:

While we are cutting over to the new designs, we need to maintain the existing `Template` and also work with the new design that is introduced in `Template_v1`  My plan for the cutover is to make it so that when `Template` is no longer being used, we remove it altogether and then possibly rename `_v1` to no longer carry this suffix. 

This is the strategy for other helper functions (like `rebuild_fail_message`) as well. 

It is cumbersome in the interim, but it's following a system of naming that I expect will unambiguously allow us to have both the old and new while we need them to coexist.

Or perhaps I am missing the point of your suggestion?

Reply:
Thanks for the clarification! The idea behind is pretty clear. I'm a bit concerned about the code cohabitation, but as I don't have a better solution to suggest, let's stick to your suggestion :+1: